### PR TITLE
HCD principles in conducting a Needs Assessment

### PR DESCRIPTION
**Pull request by mutsacc**
Fixes https://github.com/MERLCenter/merl-center/issues/104

 Patricia Rogers' depiction of the Real-Time Evaluation process for the Unstructured Data Collection methods article

### CODE REVIEW

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+layout: blog-post
+title: The use of HCD principles in conducting a needs assessment
+authors:
+  - author: Mutsa Carole Chinyamakobvu
+    authorTitle: 
+    authorGitHubHandle: mutsacc
+    authorGitHubID: 26001770
+
+tags:
+  #- tag: Data Collection and Management
```
Comment (Rabia2219) [inline _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md:11]: Hi @mutsacc,
It looks like you accidentally commented out the tag. Please fix.

**Review by Rabia2219 — CHANGES_REQUESTED** [review]

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md (hunk 1 above)
Comment (mutsacc) [reply _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md]: Thanks for picking that up @Rabia2219 - fixed.

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+The students are viewed as change agents in their home and in the community, as they are encouraged to study while also being equipped through “training to generate sustainable income, health, family cooperation, and peace at home”. This education is transferred back at home through various projects to uplift the family economically, and it transferred through educative theater, workshops and radio programmes - all which results in students as young as seventeen years old being [able to attest](https://www.youtube.com/watch?v=K2opGQ3jQJo&t=153s) that they have managed to build their family a house from revenue they are getting from family farming projects, for example. In contrast to other schools, URDT Girls’ School aims to produce “functionally literate, principled, entrepreneurial and responsible citizens”.
```
Comment (ronna) [inline _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md:64]: missing _"is"_ between it and  transferred through educative theatre...

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+The conducting of a needs assessment that the author had thought would only take a day, had taken an entire week. The week focused on getting oriented to the teaching staff’s ways of engagement and eliciting their requirements for the e-learning course, by employing various real-time evaluation techniques such as impromptu interviews, focus groups and constant observations. The observations from each day informed the course of the next, and the evaluation continued until there was sufficient information to design the required learning materials. Patricia Rogers (Better Evaluation, 2020) aptly describes real-time evaluation as a combination of the following factors: real-time data, real-time reporting, multiple cycles, double loop learning and diverse sensemaking.
+
+![](/assets/img/posts/RealTimeEvaluations_PRogers.png)
```
Comment (Rabia2219) [inline _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md:84]: @mutsacc 
Reduce size to about 70% of current. Appears very large on site.

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+Many of these steps were employed in various ways and sequences in order to determine the objectives, content, and structure of the e-learning methods course to be given to the teaching staff for the remainder of the time spent at URDT.
+
+### How we incorporated HCD
+
+The [MERL Center definition of HCD](https://merlcenter.org/human-centered-design-definition.html) is as follows:
+> “Human-centered design (HCD) is a framework that places the needs, desires and behaviors of key stakeholders, beneficiaries, users or teaching staff at the center of design and implementation decisions. HCD can be used to create and inform digital products, physical products, programs and communities. It can rely on iterative cycles of co-creating, collaborating, testing and refining solutions. HCD requires contextual analysis and understanding in order to base design decisions on how stakeholders or end-users already think, communicate and engage. In MERL, HCD can be used to inform who is asked what, when and how. It can provide insights on why something is happening. In using HCD for MERL, it is important to consider human rights and equitable solutions.” <
```
Comment (Rabia2219) [inline _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md:95]: @mutsacc 
Remove the extra '<' at the end of the definition.
Add a line break after the definition.

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+[Uganda Rural Development Training (URDT)  Girls’ School](https://urdt.net/urdt-girls-school-urdtgs/), is an all girl’s school that was established in 2000 in Kagadi district in Western Uganda. The school was instituted to “fill the need for education that is linked to development and women empowerment”. Their pedagogical approach is to employ the 2-generation approach, where parents learn along with their children to “develop a shared vision for their home, analyze their current situation, apply systems thinking, team learning, plan together and learn new skills”.
```
Comment (Rabia2219) [inline _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md:62]: @mutsacc 
Please check the link. Error: 'This site can’t be reached'

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+
+It was probable to the author that cultural differences between how URDT teaching staff and the visiting lecturers engaged, taught, learned, communicated, and prioritized activities - made a typical data collection process for the needs inadequate. Often, assessment data collection tools consist of online or paper surveys and interviews that are designed in light of the existing available data. Obtaining available data was a substantial challenge in an agriculture-focused institution where teaching, assessment and communication methods did not prioritize written accounts or data storage in ways common to the visiting lecturers. The teaching staff’s experience with tech tools was non-uniform and sparse. Their teaching [methodology](https://urdt.net/our-methodology/) was one that prioritized group discussions and practical hands-on work to enable swift rural transformation. As such, it was decided to run focus group interviews in order to get as much information from the variation of responses given, and account for the gaps in hearing and understanding that had been identified earlier in the interactions of  the volunteer lecturers and URDT staff.
```
Comment (Rabia2219) [inline _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md:66]: @mutsacc 
Please check this link https://urdt.net/our-methodology/.  Error: 'This site can’t be reached`

**Review by Rabia2219 — CHANGES_REQUESTED** [review]

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+The conducting of a needs assessment that the author had thought would only take a day, had taken an entire week. The week focused on getting oriented to the teaching staff’s ways of engagement and eliciting their requirements for the e-learning course, by employing various real-time evaluation techniques such as impromptu interviews, focus groups and constant observations. The observations from each day informed the course of the next, and the evaluation continued until there was sufficient information to design the required learning materials. Patricia Rogers (Better Evaluation, 2020) aptly describes real-time evaluation as a combination of the following factors: real-time data, real-time reporting, multiple cycles, double loop learning and diverse sensemaking.
+
+![](/assets/img/posts/RealTimeEvaluations_PRogers_v2.png)
```
Comment (Rabia2219) [inline _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md:84]: @mutsacc 
Please check image name to match with what is linked on the site.
`assets/img/posts/RealTime Evaluations_PRogers_v2.jpg`

File: _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+> “Human-centered design (HCD) is a framework that places the needs, desires and behaviors of key stakeholders, beneficiaries, users or teaching staff at the center of design and implementation decisions. HCD can be used to create and inform digital products, physical products, programs and communities. It can rely on iterative cycles of co-creating, collaborating, testing and refining solutions. HCD requires contextual analysis and understanding in order to base design decisions on how stakeholders or end-users already think, communicate and engage. In MERL, HCD can be used to inform who is asked what, when and how. It can provide insights on why something is happening. In using HCD for MERL, it is important to consider human rights and equitable solutions.”
+
+
+In an attempt to align with this definition, it is necessary to note that, not only was it important to figure out what the teaching staff needed in order to continue teaching when students are not physically at school, it was also important to figure out how to elicit accurate and relevant information for an effective course delivery.
```
Comment (Rabia2219) [inline _caseStudies/2023-05-10-hcd-principles-in-conducting-a-needs-assessment.md:98]: @mutsacc 
Please see screenshot. There needs to be a space between the grey box and the next paragraph starting with ` In an attempt.`
<details>
  <summary>Screenshot</summary>
   

</details>

**Review by Rabia2219 — CHANGES_REQUESTED** [review]

**Comment by Rabia2219** [comment]
@MattPereira 
Hi Matt,
Even though Mutsa has apparently resolved the issues on line 84 and 99 (files changed), I am not seeing them on my localhost. Specifically, I am getting the following error for the image `ERROR `/assets/img/posts/RealTimeEvaluations_PRogers_v2.png' not found.` and not seeing the space between the grey box and the next line. (as shown in screenshot)
 Can you please verify that they are actually resolved? If you can see the changes, then what is wrong at my end?

**Comment by MattPereira** [comment]
@Rabia2219 

with markdown syntax you either have to add `\` or a `<br>` to force space after a block-quote

adding extra newlines with the return key has no effect on spacing in this scenario

https://chat.openai.com/share/a71e01f2-ff5b-4a2d-9b84-83be556b3184

**Comment by Rabia2219** [comment]
> @Rabia2219
> 
> with markdown syntax you either have to add a `\` or a `<br>` to force space after a block-quote
> 
> adding extra newlines with the return key has no effect on spacing in this scenario
> 
> https://chat.openai.com/share/a71e01f2-ff5b-4a2d-9b84-83be556b3184
> 
> I think the image issue has to do with the name of the file not matching. There are three versions of the same image and some of them contain a space which is probably not allowed for jekyll build process. Once the space is removed and the name of the image file precisely matches it will show the image
> 
> See screenshot
> image

Thank you, Matt. I'll have Mutsa redo the changes by following your feedback.

**Review by ExperimentsInHonesty — APPROVED** [review]
It looks like all outstanding issues have been resolved.   I approve.

**Pull request merged**